elastic/kibana
# [Fleet] Show multiple agent policies in integrations table

Closes https://github.com/elastic/kibana/issues/182111

## Summary
 Show multiple agent policies in integrations table.

## Testing
- Enable feature flag `enableReusableIntegrationPolicies`
- Install an integration that has more than one agent policies associated with it (instructions are [here](https://github.com/elastic/kibana/pull/185916))
- Navigate to integrations table and verify that the policy displays a badge with the number of associated policies -1 and that it opens up a popover, like in below screenshots.

**NOTE** the button "Manage agent policies" does not work for now, as the feature is under development and it's hidden with a feature flag!

### With feature flag enabled, when integration has multiple agent policies

![Screenshot 2024-06-12 at 15 49 34](https://github.com/elastic/kibana/assets/16084106/bd8a4e6a-a752-46bb-8003-a4e875d0fa93)

![Screenshot 2024-06-12 at 15 46 38](https://github.com/elastic/kibana/assets/16084106/f93a91bc-bae7-40a0-8425-ac2dbbcaeae4)

When one of the policies is managed:
![Screenshot 2024-06-13 at 11 30 01](https://github.com/elastic/kibana/assets/16084106/3ba0d5cb-4af1-46e5-875a-d4391c79ad6d)


### When feature flag not enabled or integration has only one agent policy

The UI remains as it is today:
![Screenshot 2024-06-12 at 15 48 56](https://github.com/elastic/kibana/assets/16084106/14122ad3-d4f8-448a-b4b3-6f08900ba833)


### Checklist
